presto-docs/src/main/sphinx/functions/setdigest.rst

```diff
@@ -0,0 +1,132 @@
+====================
+Set Digest functions
+====================
+
+MinHash, or the min-wise independent permutations locality-sensitive hashing scheme,
+is a technique used in computer science to quickly estimate how similar two sets are.
+MinHash serves as a probabilistic data structure that estimates the Jaccard similarity
+coefficient - the measure of the overlap between two sets as a percentage of the total unique elements in both sets.
+Presto offers several functions that deal with the
+`MinHash <https://wikipedia.org/wiki/MinHash>`technique.
+
+MinHash is used to quickly estimate the
+`Jaccard similarity coefficient <https://wikipedia.org/wiki/Jaccard_index>`
+between two sets.
+It is commonly used in data mining to detect near-duplicate web pages at scale.
+By using this information, the search engines efficiently avoid showing
+within the search results two pages that are nearly identical.
+
+In PrestoDB, set digest functions are a series of functions designed to create a theta sketch for specific set operations.
+Set digest functions can be considered a probabilistic data structure. These functions are useful for operations
+like union, intersection, and difference of sets, estimation of set sizes, and so forth.The primary set digest function
+in PrestoDB is setdigest(). It combines values into a theta sketch, which is a type of
+data sketch used in approximating the total distinct count of a dataset.
+
+Here's how you use it:
+SELECT cardinality(setdigest(value)) FROM table
+
+This generates a theta sketch and estimates the cardinality (distinct count) of the 'value' column in the 'table'.
+
+Data structures
+---------------
+
+Presto implements Set Digest data sketches by encapsulating the following components:
+
+- `HyperLogLog <https://wikipedia.org/wiki/HyperLogLog>`
+- `MinHash with a single hash function <http://wikipedia.org/wiki/MinHash#Variant_with_a_single_hash_function>`
+
+As of now, HyperLogLog and MinHash are among the techniques implemented in Presto or used
+by certain functions in Presto to handle large data sets.
+HyperLogLog (HLL): HyperLogLog is an algorithm used to estimate the cardinality
+of a set — that is, the number of distinct elements in a large data set.
+Presto uses it to provide the function approx_distinct which can be used to estimate the number
+of distinct entries in a column.
+
+MinHash: MinHash is used to estimate the similarity between two or more sets, commonly known as Jaccard similarity.
+It is particularly effective when dealing with large data sets and is generally used in data clustering
+and near-duplicate detection.
+
+In the future, Presto may incorporate more sketching techniques or data structures for handling large
```

Comment:
In general, documentation tends against forward-looking or future statements, and tries to focus on the existing state of the code or product. Because of this, I suggest deleting this paragraph.